docker/admin.air.toml

```diff
@@ -0,0 +1,46 @@
+root = "."
```

Comment:
I don't think we need this file? We already have the `docker/.admin.air.toml` - That takes care of running the admin app. Are there any specific settings you need in this file that we don't already have in the `.admin.air.toml`?